src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -961,6 +961,121 @@ def test_configure_pattern_burst_sites(multi_instrument_session):
+    configure_session(multi_instrument_session, test_name)
+    multi_instrument_session.load_pattern(get_test_file_path(test_name, 'pattern.digipat'))
+
+    multi_instrument_session.pins['LO0'].create_source_waveform_serial(
+        waveform_name='src_wfm',
+        data_mapping=nidigital.SourceDataMapping.BROADCAST,
+        sample_width=2,
+        bit_order=nidigital.BitOrder.LSB)
+
+    # load and burst the waveform to confirm that configuration went okay
+    multi_instrument_session.write_source_waveform_broadcast(
+        waveform_name='src_wfm',
+        waveform_data=[i for i in range(4)])
+    pass_fail = multi_instrument_session.burst_pattern(start_label='new_pattern')
+    assert pass_fail == {0: True, 1: True}
+
+
+def test_create_source_waveform_from_file_tdms(multi_instrument_session):
+    test_name = 'test_source_waveform_parallel_broadcast'
+    configure_session(multi_instrument_session, test_name)
+    multi_instrument_session.load_pattern(get_test_file_path(test_name, 'pattern.digipat'))
+
+    multi_instrument_session.create_source_waveform_from_file_tdms(
+        waveform_name='src_wfm',
+        waveform_file_path=get_test_file_path(test_name, 'source_waveform.tdms'),
+        write_waveform_data=True)
+
+    # burst the waveform to confirm that configuration and loading went okay
+    pass_fail = multi_instrument_session.burst_pattern(start_label='new_pattern')
+    assert pass_fail == {0: True, 1: True}
+
+
+def test_write_source_waveform_data_from_file_tdms(multi_instrument_session):
+    test_name = 'test_source_waveform_parallel_broadcast'
+    configure_session(multi_instrument_session, test_name)
+    multi_instrument_session.load_pattern(get_test_file_path(test_name, 'pattern.digipat'))
+
+    multi_instrument_session.create_source_waveform_from_file_tdms(
+        waveform_name='src_wfm',
+        waveform_file_path=get_test_file_path(test_name, 'source_waveform.tdms'),
+        write_waveform_data=False)
+    try:  # confirm that the waveform is not yet loaded
+        multi_instrument_session.burst_pattern(start_label='new_pattern')
+        assert False
+    except nidigital.Error as e:
+        assert e.code == -1074118614
+        assert e.description.find('The source waveform(s) used in the pattern(s) to be burst have not been written to source memory.'
+                                  ' Ensure that you write source waveforms with niDigital Write Source Waveform.') != -1
+
+    multi_instrument_session.write_source_waveform_data_from_file_tdms(
+        waveform_name='src_wfm',
+        waveform_file_path=get_test_file_path(test_name, 'source_waveform.tdms'))
+
+    # burst the waveform to confirm that configuration and loading went okay
+    pass_fail = multi_instrument_session.burst_pattern(start_label='new_pattern')
+    assert pass_fail == {0: True, 1: True}
+
+
+def test_create_capture_waveform_serial(multi_instrument_session):
+    test_name = 'test_fetch_capture_waveform'
```

Comment:
Rename'test_fetch_capture_waveform' to 'test_fetch_capture_waveform_parallel' and create a new project - 'test_fetch_capture_waveform_serial' with the same souce configuration as that in test_create_source_waveform_serial.

Use this pattern:
![image](https://user-images.githubusercontent.com/48041236/80335712-c42c8280-881a-11ea-928d-f003904c1a8a.png)

Capture configuration: LSB, 2 bits, and HI0.

Reply:
Done, though I named the new project 'test_create_source_waveform_serial'.
I updated the test to use your recommended capture configuration, in addition to creating that configuration within the new project.